src/main/scala/org/holmesprocessing/totem/services/pemeta/Dockerfile

```diff
@@ -0,0 +1,50 @@
+FROM golang:alpine
+
+# Create folder
+RUN mkdir -p /service
+WORKDIR /service
+
+# Get Golang and PEV dependencies
+RUN apk add --no-cache \
+        git \
+        && go get github.com/julienschmidt/httprouter \
+        && git clone -b testing https://github.com/boddumanohar/libpe.git \
+        && rm -rf /var/cache/apk/*
+
+RUN apk add --no-cache \
+        bash \
+        build-base \
+            openssl-dev \
+	    make \
+	    g++ \
+        && rm -rf /var/cache/apk/*
+
+# Clean Up
+RUN apk del --purge \
+        build-base \
+        && rm -rf /var/cache/apk/* yara-3.5.0
```

Comment:
you are removing yara-3.5.0 but have not installed it. Looks like a copy over from a previous file you were working on.